app/src/main/java/com/example/placefinder/MainViewModel.kt

```diff
@@ -8,7 +8,7 @@ import com.example.placefinder.service.ParkService
 import kotlinx.coroutines.launch
 
 class MainViewModel: ViewModel() {
-    var parks = MutableLiveData<List<Park>>()
+    var parks = MutableLiveData<List<Park>?>()
```

Comment:
Is there a reason where this will be null?  If not, best to make it a not-nullable type, to avoid potential NullPointerException.